### ISSUE TITLE
Search function looks cluttered

### ISSUE THREAD

**Issue by MrTengil** [tskarhed/setlist#13]
### Using the search function is confusing and not intuitive.
I suggest something like this:
![image](https://user-images.githubusercontent.com/52031625/82732197-f3e08480-9d0b-11ea-9eaf-814cfcf22a0b.png)
This gives the user a better visual feedback that they are searching.

The style, shapes and colors are tbd but you get the gist!

**Also!**
Is it used to search for songs or setlists?

![image](https://user-images.githubusercontent.com/52031625/82731915-415bf200-9d0a-11ea-87de-27fa04fdc5d0.png)

Setlists and songs looks too similar in the search. They need to be visually seperated more.
Maybe add a horizontal divider between songs and setlist?
Bold the setlists?
Italic on the songs?

I assume there will be more songs than setlists. Therefore the setlists should be presented before the songs. This will ensure that the user does not have to scroll too much to find the item.

**Comment by tskarhed** [comment]
Animation, sure, just want to get #6 done, otherwise the code is going to be very messy.

Search is for both setlists and songs, a sort of search first navigation. The order doesn't matter that much, but I would personally have 3 most recent first. The main page already lists all setlists, so they should be low prio. Are the icons so separate the types unclear?

**Comment by tskarhed** [comment]
Search should be available from all pages, to easy jump between songs without complex navigation.

**Comment by MrTengil** [comment]
What about hamburger menu?

**Comment by tskarhed** [comment]
Nah, I thing just a bottom navigation would make sense, a bit like Spotify.
